frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -1,38 +1,52 @@
 import React, { useState } from 'react';
 import map from 'lodash/map';
+import * as moment from 'moment';
```

Comment:
It may seem crazy to worry about it, but I wonder if it'd be better to use [date-fns](https://date-fns.org/) instead of moment. Moment is.. pretty bulky.

Reply:
Will check out and try to implement date-fns!